packages/neon/neon_talk/lib/l10n/en.arb

```diff
@@ -18,6 +18,19 @@
   "roomMessageReply": "Reply",
   "roomMessageReaction": "Add reaction",
   "roomMessageDelete": "Delete",
+  "roomMessageEdit": "Edit",
+  "roomMessageEdited": "edited",
+  "roomMessageLastEdited": "Last edited by {name} at {time}",
+  "@roomMessageLastEdited": {
+    "placeholders": {
+      "name": {
+        "type": "String"
+      },
+      "time": {
+        "type": "String"
+      }
+    }
+  },
```

Comment:
I'm not sure if you are aware of this, but `intl` can also format dates.

```suggestion
  "roomMessageEdit": "Edit",
  "roomMessageEdited": "edited",
  "roomMessageLastEdited": "Last edited by {name} at {time}",
  "@roomMessageLastEdited": {
    "placeholders": {
      "name": {
        "type": "String"
      },
      "time": {
        "type": "DateTime",
        "format": "yMd"
      }
    }
  },
```

Reply:
In this case I need `yMd` and `jm`, but there is no constructor for that. Otherwise this would have been a nice option.